src/styles/card.scss

```
@@ -0,0 +1,59 @@
+.card {
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  padding-inline: 16px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+  width: 166px;
+  font-family: $font-family;
+  background-color: $bg-color;
+
+  &__monitor {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin-inline: 3px;
+    margin-bottom: 40px;
+  }
+
+  &__name {
+    margin: 0;
```

reset margin for all `p` and try to use `margin-bottom` to make outer indent between elements
```suggestion
    margin-bottom: 4px;
```